chcemvediet/apps/accounts/forms.py

```diff
@@ -99,10 +100,13 @@ class SettingsForm(forms.Form):
             )
 
     days_to_publish_inforequest = forms.IntegerField(
-            min_value=0,
-            max_value=MAX_DAYS_TO_PUBLISH_INFOREQUEST,
```

Comment:
Pozor, min/max value potrebujes nechat aj na fielde, koli backend validacii. Nech sa limity nekontroluju iba na FE. Inac ked niekto formular submitne rucne, tak nam tam savne cokolvek.

Skus si v browsri otvorit development konzolu a vykonany POST request rucne zopakovat s inymi hodnotami. Ak pouzijes hodnoty mimo min/max value, tak BE by mal skoncit chybou.

Reply:
pri POST request `...&days_to_publish_inforequest=12345` browser chybou neskoncil ale do databazy sa ulozila hodnota `12`. Skusal som viacero pokusov a z daneho cisla zobralo len jeho zaciatok, ale nikdy neulozilo hodnotu mimo rozsah.

Reply:
respektive sa to sprava nepravidelne, nikedy dany requuest pri zlych hodnotach `days_to_publish_inforequest` sa nevykona vobec.

Reply:
@viliambalaz Neviem ako si skusal, ake ked som nechal `max_value=MAX_DAYS_TO_PUBLISH_INFOREQUEST` a docasne upravil slider na FE zmenou atributu `max` na 1000, tak po submitnuti hodnoty vacsej nez `MAX_DAYS_TO_PUBLISH_INFOREQUEST` to skoncilo pekne chybovou hlaskou:

![max](https://user-images.githubusercontent.com/5547275/81668963-b8a9a000-9445-11ea-9890-32c4931b9fb1.png)

Samozrejme normalne sa ta chybova hlaska userovi nezobrazi. Ake **kazda** FE validacia sa **musi** validovat aj na BE.

Reply:
> respektive sa to sprava nepravidelne, nikedy dany requuest pri zlych hodnotach `days_to_publish_inforequest` sa nevykona vobec.

Tipujem, ze si posielal nevalidne requesty na BE. Najkor ti nesedela hodnota v HTTP hlavicke `Content-Length`. Lebo ked si POST data zeditoval, zrejme mali inu dlzku.

Reply:
:ok: